lib/driver/sdr_driver.c

```diff
@@ -88,3 +91,13 @@ os_task_return_t sdr_rx_task(void *param) {
         }
     }
 }
+
+int sdr_uhf_set_rf_mode(uint8_t rf_mode){
+    csp_iface_t * iface = csp_iflist_get_by_name(SDR_IF_UHF_NAME);
```

Comment:
Has this been tested on ground station? It's pretty picky about things

Reply:
Just tested it; works fine